### ISSUE TITLE
Diagnostics for Training Set

### ISSUE THREAD

**Issue by annayqho** [annayqho/TheCannon#5]
# Current Plots:
- Plot the coverage of label space in the training set (log x-axis)
- Histograms of training & test S/N, overplotted
- Auto-correlation of residual spectrum (if the residual is actually dominated by noise, should get a delta function)

**Comment by annayqho** [comment]
How should we quantify whether the S/N or set size is "not big enough"?

**Comment by annayqho** [comment]
Fit a Gaussian to each S/N distribution, and compare the mean. If mean(S/N training set) < mean(S/N test set), print a warning?

**Comment by davidwhogg** [comment]
I don't see why this warning is necessary or relevant to anything; explain here or by email.
